docs/source/operators/ipython_security.asc

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Added by accident?

Reply:
this is added because security.rst says `If you prefer to encrypt your security reports,
you can use :download:`this PGP public key <ipython_security.asc>`.`

Reply:
As discussed, it should be there.